holopy/scattering/interface.py

```diff
@@ -195,17 +193,8 @@ def calc_holo(detector, scatterer, medium_index=None, illum_wavelen=None,
     theory = interpret_theory(scatterer, theory)
     uschema = prep_schema(
         detector, medium_index, illum_wavelen, illum_polarization)
-
```

Comment:
ZOMG, did you eliminate the need for this ugly massaging of scaling with color channels??!!!?! If so, :100:

Reply:
It is now taken care of by `scatteringtheory.py` and other places. :100: